src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -576,6 +576,8 @@ def _job_cluster_spark_conf(self, cluster_key: str):
             return {"spark.sql.sources.parallelPartitionDiscovery.parallelism": "200"} | conf_from_installation
         return conf_from_installation
 
+    # Workflow creation might fail on an InternalError with no message
+    @retried(on=[InternalError], timeout=timedelta(minutes=2))
```

Comment:
@nfx : Could you introduce me to the right people to raise the `InternalError: None` with job creation with?